src/main/java/io/aiven/kafka/connect/common/grouper/RecordGrouper.java

```diff
@@ -42,4 +42,10 @@
      */
     Map<String, List<SinkRecord>> records();
 
+    interface Rotator<T> {
```

Comment:
Do we expect any other type except List of Sink Records?

Reply:
We expect both

Reply:
Can we narrow down a requirement for T type somehow?